src/components/container/MostRecentNotification/MostRecentNotification.tsx

```
@@ -38,31 +62,67 @@
+			count++;
+			const result = await MyDataHelps.queryNotifications(queryParams);
+			const matchingNotification = result.notifications.find(n => n.identifier.match(identifierRegex));
+			if (matchingNotification) {
+				return matchingNotification;
+			}
+			if (!result.nextPageID) {
+				return null;
+			}
+			queryParams.pageID = result.nextPageID;
+
+		} while (true);
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)